### PR TITLE
Improvements to the Crowdin presence

### PR DESCRIPTION
**Pull request by ImRodry**
Removed `@types/ora` as this was deprecated on version 3.2.0 of ora
Fixed unhandled errors with a project's main page and profiles with no nickname (username = display name)
Fixed profile nickname sometimes being the job title instead
Removed double spacing when translating a project
Optimized browsing timestamp
Optimized query selectors overall
Made variable types more accurate
Added an option when viewing a profile that checks if it's the user's own profile.
Remove project/translators tab as this was removed months ago and is now only accessible to managers under the project settings

This is a somewhat comprehensive list of the fixes and additions. If you need additional proof let me know.
image
image
Users with no nickname and a job:
Before: 
image
After: 
image
New project presence style: (Project name / tab)
image

### CODE REVIEW

File: websites/C/Crowdin/dist/metadata.json [hunk 1]
```diff
@@ -3,6 +3,12 @@
     "name": "TheDrop.",
     "id": "293828021134295040"
   },
+  "contributors": [
+    {
+      "name": "Rodry",
+      "id": "240875059953139714"
+    }
+  ],
```
Comment (ririxidev) [inline websites/C/Crowdin/dist/metadata.json:11]: Changes not worth adding you as contributor.

**Review by ririxidev — CHANGES_REQUESTED** [review]
and revert changes to package.json & yarn.lock. I will push it myself later

File: websites/C/Crowdin/dist/metadata.json (hunk 1 above)
Comment (ImRodry) [reply websites/C/Crowdin/dist/metadata.json:11]: I literally changed the entire presence wdym lol

**Comment by ImRodry** [comment]
I was told to remove the package myself on this PR by @Bas950 so you should probably talk to him

**Comment by ririxidev** [comment]
Just remove it - i will push it later with some changes.

File: websites/C/Crowdin/dist/metadata.json (hunk 1 above)
Comment (ririxidev) [reply websites/C/Crowdin/dist/metadata.json:11]: Not really

**Comment by ImRodry** [comment]
> Just remove it - i will push it later with some changes.

Again, Bas is in a higher position than you so I will trust his judgment until told otherwise.

File: websites/C/Crowdin/dist/metadata.json (hunk 1 above)
Comment (ririxidev) [reply websites/C/Crowdin/dist/metadata.json:11]: but ok

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**